minho/programmers/Level_1/7일차/신규_아이디_추천.js

```diff
@@ -0,0 +1,22 @@
+function solution(new_id) {
+    new_id = new_id
+        .toLowerCase() // 1단계
+        .replace(/[~!@#$%^&*()=+:?,<>/\[\]\{\}]/g, '') // 2단계
+        .replace(/\.{2,}/g, '.') // 3단계
+        .replace(/\.$|^\./g,''); // 4단계
+    console.log(new_id)
+    
+    if(!new_id){ // 5단계
+        new_id = 'a'; 
+    } 
+    if(new_id.length > 15){ // 6단계
+        new_id = (new_id
+            .substr(0, 15))
+            .replace(/\.$|^\./g,''); // 4단계 반복
+    }
+    if(new_id.length < 3){ // 7단계
+        let l_el = new_id.substr(-1);
```

Comment:
```suggestion
        let l_el = new_id.substr(-1);
```
l_el 이라는 변수명은 무슨 뜻인가요?

Reply:
last_el이었습니다 다음에는 변수명 더 명확하게 할게요!